sw/rust/demo/lcd_hal/src/main.rs

```diff
@@ -23,8 +24,7 @@ use st7735_lcd::Orientation;
 
 use crate::hal::{pac, GpioExt};
 
-use embedded_hal;
-use embedded_hal::{blocking::delay::DelayMs, digital::v2::OutputPin};
+use embedded_hal::{self, delay::DelayNs, digital::OutputPin};
```

Comment:
Does the `DelayNs` package still have the `delay_ms` function?

Reply:
Yes.
https://docs.rs/embedded-hal/1.0.0/embedded_hal/delay/trait.DelayNs.html